src/components/v5/common/ActionSidebar/ActionTypeSelect.tsx

```diff
@@ -103,16 +101,19 @@ const ActionTypeSelect: FC<ActionTypeSelectProps> = ({ className }) => {
                     return;
                   }
 
-                  if (
-                    Object.keys(formState.dirtyFields).length > 0 &&
-                    actionType
-                  ) {
+                  const hasMadeChanges = Object.keys(
+                    formState.dirtyFields,
+                  ).find(
+                    (fieldName) =>
+                      fieldName !== 'title' && fieldName !== 'actionType',
+                  );
+
+                  if (hasMadeChanges && actionType) {
```

Comment:
It's a bit manual now that we have to specify the "common" fields here, and then make sure we're also setting them in `onConfirm`. Could those be stored in a config somewhere that we could iterate over?